src/docs/arch-timestamp.dox

```diff
@@ -209,9 +212,17 @@ for aggregating time windows and merging with other time aggregates;
 this is used during reconciliation to accumulate the aggregate value
 to be stored in the parent.
 (Otherwise, it was not, and those operations are prohibited.)
+
```

Comment:
FWIW, I left a comment (post merge) for @sauclovian-wt on WT-7563 suggesting that it might be helpful to define the aggregated time window at a higher level than just listing the fields.  I.e., something like "it is the smallest time window that includes the time windows of all of the records on the corresponding page or subtree".

Reply:
oh! I misunderstood what you wanted, will try again.

Reply:
(for the record, this change got merged here in d1e06e0)